### PR TITLE
Fix logic for SSL certificate verification in OpenShift scans

### PR DESCRIPTION
**Pull request by bruno-fs**
- OCP Scanner should now respect SSL options (disable and verify)
- Added the ability to control timeouts for OCP scans via env vars `QPC_CONNECT_TASK_TIMEOUT` and `QPC_INSPECT_TASK_TIMEOUT` (in seconds)

### CODE REVIEW

**Review by nicolearagao — APPROVED** [review]
LGTM 👍🏼

File: quipucords/api/scantask/model.py [hunk 1]
```diff
@@ -213,7 +212,7 @@ def log_message(
         """Log a message for this task."""
         # pylint: disable=no-member
         if exception:
-            logger.error(traceback.format_tb(exception.__traceback__))
+            logger.exception("Got an exception.")
```
Comment (gtanzillo) [inline quipucords/api/scantask/model.py:215]: Won't we lose what the exception is, in addition to removing the traceback? Or does `logger.exception` handle that?

Comment (bruno-fs) [reply quipucords/api/scantask/model.py:215]: logger.exception handles that =D
turns out that formatting in the old way was losing a lot of context from the traceback.

This is how it looks like now
image

**Review by gtanzillo — APPROVED** [review]
LGTM 👍

**Pull request merged**